android/uhabits-android/src/main/java/org/isoron/uhabits/widgets/activities/NumericalCheckmarkWidgetActivity.kt

```diff
@@ -0,0 +1,69 @@
+package org.isoron.uhabits.widgets.activities
+
+import android.app.Activity
+import android.appwidget.AppWidgetManager
+import org.isoron.uhabits.R
+import android.content.Context
+import android.os.Bundle
+import android.view.Window
+import org.isoron.androidbase.AppContextModule
+import org.isoron.uhabits.HabitsApplication
+import org.isoron.uhabits.activities.common.dialogs.NumberPickerFactory
+import org.isoron.uhabits.core.ui.screens.habits.list.ListHabitsBehavior
+import org.isoron.uhabits.core.ui.widgets.NumericalCheckmarkWidgetBehavior
+import org.isoron.uhabits.intents.IntentFactory
+import org.isoron.uhabits.intents.IntentParser
+import org.isoron.uhabits.intents.PendingIntentFactory
+import org.isoron.uhabits.widgets.WidgetUpdater
+
+class NumericalCheckmarkWidgetActivity : Activity() {
+
+    private lateinit var behavior : NumericalCheckmarkWidgetBehavior
+    private var data: IntentParser.CheckmarkIntentData? = null
+    private lateinit var widgetUpdater : WidgetUpdater
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        requestWindowFeature(Window.FEATURE_NO_TITLE)
+        setContentView(R.layout.widget_checkmark_number_screen)
```

Comment:
Is this any different from `setContentView(FrameLayout(this))`? If there's no difference, please remove the layout file.